pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
@dk1a did you do anything in particular to force the lockfile version? when I try to `pnpm install` or `pnpm install -- force`, I still get the old version, even though I'm on pnpm 8.6.2

Reply:
even clearing out `node_modules` and reinstalling doesn't seem to help

Reply:
no, pnpm sometimes suggests upgrading itself, so I just did `pnpm add -g pnpm`,
I actually had a reverse problem where I couldn't really get '6.0'
you could try removing pnpm and installing it again I guess

@alvrs @roninjin10 curious about your experience with lockfileVersion

Reply:
Npm uninstall and reinstall pnpm is another thing to try.   Also if you use nvm or anything where you have more than one node version might need to check specific node versions global store

Reply:
I tried `pnpm store prune`, removing `pnpm-lock.yaml` and reinstalling, uninstalling `pnpm` and reinstalling, and still no luck.

Reply:
turns out it was this: 
![image](https://github.com/latticexyz/mud/assets/508855/5f4b7a6c-2312-4ba7-b1bf-65b04200828e)

Reply:
Ah glad it was something that actually makes sense.   I was about to quit using pnpm when you mentioned reinstalling pnpm doesn't fix it.  

As an aside it's interesting how often pnpm changes their schema version compared to yarn which never changes it